data_steward/cdr_cleaner/cleaning_rules/unit_normalization.py

```diff
@@ -163,10 +164,16 @@ def setup_rule(self, client=None):
         defined as part of get_query_specs().
         :param client:
         :return:
+
+        :raises:  BadRequest, OSError, AttributeError, TypeError, ValueError if
+            the load job fails. Error raised from bq.upload_csv_data_to_bq_table
+            helper function.
         """
 
         # creating _unit_mapping table
-        unit_mapping_table = f'{self.project_id}.{self.dataset_id}.{UNIT_MAPPING_TABLE}'
+        unit_mapping_table = (f'{self.project_id}.'
+                              f'{self.sandbox_dataset_id}.'
+                              f'{UNIT_MAPPING_TABLE}')
         bq.create_tables(
```

Comment:
@lrwb-aou do we want to add exists_ok=True to create_tables for creating _unit_mapping?